google_takeout_parser/parse_json.py

```diff
@@ -252,13 +259,14 @@ def _parse_chrome_history(p: Path) -> Iterator[Res[ChromeHistory]]:
         yield RuntimeError(f"Chrome/BrowserHistory: no 'Browser History' key in '{p}'")
     for item in json_data.get("Browser History", []):
         try:
-            time_naive = datetime.utcfromtimestamp(item["time_usec"] / 10**6)
+            time_naive = datetime.utcfromtimestamp(item.get("time_usec") / 10**6)
```

Comment:
same here, this leads to a `None / 10**6` which would just crash, should just direct access